lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2170,18 +2170,32 @@ TypeSystemSwiftTypeRef::GetBitSize(opaque_compiler_type_t type,
       // info for it, so defer to SwiftASTContext.
       if (llvm::isa<SwiftASTContextForExpressions>(m_swift_ast_context))
         return ReconstructType({this, type}).GetBitSize(exe_scope);
+      LLDB_LOGF(GetLogIfAllCategoriesSet(LIBLLDB_LOG_TYPES),
+                "Couldn't compute size of type %s using SwiftLanguageRuntime.",
+                AsMangledName(type));
+      return {};
     }
 
     // If there is no process, we can still try to get the static size
     // information out of DWARF. Because it is stored in the Type
     // object we need to look that up by name again.
     if (TypeSP type_sp = LookupTypeInModule(type)) {
-      if (auto byte_size = type_sp->GetByteSize(exe_scope))
+      struct SwiftType : public Type {
+        /// Avoid a potential infinite recursion because
+        /// Type::GetByteSize() may call into this function again.
+        llvm::Optional<uint64_t> GetStaticByteSize() {
```

Comment:
Thoughts on adding this function directly to `Type`?

Reply:
I did it in this odd way is to minimize the diff between swift-lldb and llvm.org lldb.